.travis.yml

```diff
@@ -0,0 +1,33 @@
+language: c
+
+cache:
+  - ccache
+
+os: linux
+dist: bionic
```

Comment:
Checking the Vulkan-Loader, it seems to use xenial. I wonder if we should align. Or is it causing issues?

Reply:
That's actually a leftover from earlier versions.

I kinda want to just drop these two lines and use the Travis defaults; is that ok with you?

Reply:
I think it would make more sense to just use a similar setup to the rest of the Khronos repos.